### PR TITLE
Docs: design tutorials page

### LINKED ISSUE #1561: [content] Patterns presentation recording for Design tutorials
Tyler and Jillian to record their Patterns presentation for a new Design Tutorials page.

### PR DESCRIPTION
**Pull request by jillianhowarth**
Closes #1561 

New page in the Designing section for our Patterns presentation content.

### CODE REVIEW

**Review by dakahn — APPROVED** [review]
i approve! 🏄

File: src/pages/designing/tutorials.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Tutorials
+description:
+  The Carbon design tutorials will get you setup to start designing with Carbon
```
Comment (emyarod) [inline src/pages/designing/tutorials.mdx:4]: is setup supposed to match set up on line 13?

**Review by emyarod — APPROVED** [review]
looks good to me

File: src/pages/designing/tutorials.mdx (hunk 1 above)
Comment (jillianhowarth) [reply src/pages/designing/tutorials.mdx]: Yeah, good catch! I just pushed a fix.

**Review by janchild — APPROVED** [review]
Well done, @jillianhowarth !!

**Pull request merged**